README.md

```diff
@@ -1,5 +1,40 @@
 # iRODS C++ REST Mid-Tier API
 
+This REST API is designed to be run alongside an iRODS Server to provide an HTTP REST interface to the majority of administrative functions for an iRODS Zone.
```

Comment:
**administrative** makes it sound like the REST API only provides admin level operations.

Reply:
oh right, that's ZMT...   rewording...